public/templates/emails/categoryNotifications_post.tpl

```diff
@@ -16,7 +16,7 @@
 				<tr>
 					<td style="padding: 0px 40px; font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol; font-size: 15px; line-height: 20px; color: #555555;">
 						<h1 style="margin: 0 0 10px 0; font-family: -apple-system,BlinkMacSystemFont,Segoe UI,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol; font-size: 18px; line-height: 21px; color: #aaaaaa; font-weight: normal;">
-							<a href="{url}/user/{user.slug}">{user.name}</a> has posted a new reply to "<a href="{url}/topic/{topicSlug}">{title}</a>" in "<a href="{url}/category/{category.slug}">{category.name}</a>"
+						    [[categorynotifications:email-categorie-post,{url},{user.slug},{user.name},{topicSlug},{title},{category.slug},{category.name}]]
```

Comment:
7 inclusions is a lot.. could we merge `{url}/user/{user.slug}`, `{url}/topic/{topicSlug}`, and `{url}/category/{category.slug}` into three pairs? Then we'd only have 4 inclusions in the language string.

Reply:
Three individual language keys, that is.

Reply:
Do you mean using 3 different language keys instead of one with 7 variables?

Some languages don't follow the same noun/verb structure so sometimes it doesn't work to have 3 keys one after another like [[namespace:key1]] [[namespace:key2]] [[namespace:key2]]

Reply:
No, I mean the one language key is fine, but the parts making up the three URLs are always going to be the same, so there's no reason to break up the individual variables out.

So instead of `[[categorynotifications:email-categorie-post,{url},{user.slug},...` you could just do `[[categorynotifications:email-categorie-post,{url}/user/{user.slug},...`

Reply:
Ok i m modifying that